src/libs/plugin/manager.cpp

```diff
@@ -462,7 +462,8 @@ PluginManager::unload(const std::string& plugin_name)
       meta_plugins_.unlock();
 
     } catch (Exception &e) {
-      LibLogger::log_error("PluginManager", "Could not finalize one or more threads of plugin %s, NOT unloading plugin", plugin_name);
+      LibLogger::log_error("PluginManager", "Could not finalize one or more threads "
+													 "of plugin %s, NOT unloading plugin", plugin_name.c_str());
```

Comment:
Wrong indentation

Reply:
Good catch, thanks. It's probably because I made those modifications on the FreeBSD host without an up-to-date Emacs config. Going to fix those quickly.